app/grandchallenge/challenges/management/commands/set_challenge_request_handling_preference.py

```diff
@@ -0,0 +1,19 @@
+from django.core.management import BaseCommand
+
+from grandchallenge.challenges.models import Challenge
+from grandchallenge.core.utils.access_request_utils import (
+    AccessRequestHandlingOptions,
+)
+
+
+class Command(BaseCommand):
+    def handle(self, *args, **options):
+        challenges = Challenge.objects.all()
+        for challenge in challenges:
+            if not challenge.require_participant_review:

```

Comment:
FYI subsetting immediately to `Challenge.object.filter(require_participant_review=False)` does not work with the deprecated field, so had to do it this way.

Reply:
I think the data migration needs to be done before the deprecation. From the docs:

> Any lingering references to it in your code will return None

So, here challenge.require_participant_review will always be `None` and no challenges will be migrated.

Reply:
Oh yes, you're right!